### DISCUSSION TITLE
Add Tika and Gotenberg to paperless-ngx

### DISCUSSION THREAD

**Discussion by denes44** [tteck/Proxmox#1324] [Ideas]
It would be good if Tika and Gotenberg could be added as default, or as an option to paperless-ngx to support Office (.doc,.xls) files:
[https://docs.paperless-ngx.com/configuration/#tika](https://docs.paperless-ngx.com/configuration/#tika)

**Comment by Lucifer1590** [comment]
came here to ask the same
